tessdata.yaml

```diff
@@ -0,0 +1,108 @@
+package:
+  name: tessdata
+  version: 4.1.0
+  epoch: 0
+  description: Tesseract training data
+  copyright:
+    - license: Apache-2.0
+
+environment:
+  contents:
+    packages:
+      - busybox
+      - curl-dev
+      - git
+
+data:
+  - name: langs
+    items:
+      afr:
+      ara:
+      aze:
+      bel:
+      ben:
+      bul:
+      cat:
+      ces:
+      chi_sim:
+      chi_tra:
+      chr:
+      dan:
+      deu:
+      eng:
+      enm:
+      epo:
+      equ:
+      est:
+      eus:
+      fin:
+      fra:
+      frk:
+      frm:
+      glg:
+      grc:
+      heb:
+      hin:
+      hrv:
+      hun:
+      ind:
+      isl:
+      ita:
+      ita_old:
+      jpn:
+      kan:
+      kat:
+      khm:
+      kor:
+      lav:
+      lit:
+      mal:
+      mkd:
+      mlt:
+      msa:
+      nld:
+      nor:
+      osd:
+      pol:
+      por:
+      ron:
+      rus:
+      slk:
+      slv:
+      spa:
+      spa_old:
+      sqi:
+      srp:
+      swa:
+      swe:
+      tam:
+      tel:
+      tgl:
+      tha:
+      tur:
+      ukr:
+      vie:
+
+pipeline:
+  - uses: git-checkout
+    with:
+      repository: https://github.com/tesseract-ocr/tessdata
+      tag: ${{package.version}}
+      expected-commit: 4767ea922bcc460e70b87b1d303ebdfed0897da8
+
+subpackages:
+  - name: tesseract-dev
+    pipeline:
+      - uses: split/dev
```

Comment:
Isn't this will create two packages with the name `tesseract-dev`. one here and another in the tesseract package yaml.

We don't need it here i guess.